core/src/main/resources/org/apache/spark/ui/static/scroll-button.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* global $ */
+
+export { addScrollButton };
+
+function addScrollButton() {
+  const container = $('<div></div>', {
+    'class': 'scroll-btn-container'
+  });
+
+  const topBtn = $('<div></div>', {
+    'class': 'scroll-btn-half scroll-btn-top'
+  }).click(function () {
+    $('html, body').animate({scrollTop: 0}, 'slow');
+  });
+
+  const bottomBtn = $('<div></div>', {
```

Comment:
Perhaps we can define some functions to improve code reusability, like 
```
function scrollToTop() {
  window.scrollTo({ top: 0, behavior: 'smooth' });
}

function scrollToBottom() {
  window.scrollTo({ top: document.documentElement.scrollHeight, behavior: 'smooth' });
}

function createScrollBtn(className, handler) {
  return $('<div>', { class: `${config.btnClass} ${className}` })
    .on('click', function(e) {
      handler();
      $(this).addClass(config.activeClass)
             .delay(300)
             .queue(() => $(this).removeClass(config.activeClass).dequeue());
    });
}
```

then 

```
const topBtn = createScrollBtn(config.topBtnClass, scrollToTop);
 const bottomBtn = createScrollBtn(config.bottomBtnClass, scrollToBottom);
```